linkerd/app/integration/src/tests/telemetry/tls_detect.rs

```diff
@@ -3,6 +3,8 @@ use std::time::SystemTime;
 
 const TIMEOUT: Duration = Duration::from_millis(640); // 640ms ought to be enough for anybody.
 
+const METRIC: &str = "inbound_tcp_accept_errors_total";
```

Comment:
nit: we should probably also rename this module from `tls_detect.rs` to `tcp_accept.rs`?